### DISCUSSION TITLE
Client wants to show stars rating on smartbanner

### DISCUSSION THREAD

**Discussion by wcc-crystal** [ain/smartbanner.js#550] [General]
My client is wanting to see how it looks with stars, and I was wondering if anyone else has tried to add that to the smartbanner. I am going to attempt to try, but I do not know exactly where to start yet as I just installed smartbanner this hour.

**Comment by ain** [comment]
The stars were discussed in #50 and the support will come, but the work has not yet been started. Contributions are therefore more than welcome!
